pubspec.yaml

```diff
@@ -7,15 +7,17 @@ description: >-
 
 homepage: https://github.com/kiwi-bop/intl_generator
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ">=2.17.0 <3.0.0"
 
 dependencies:
   collection: ">=1.15.0 <2.0.0"
-  analyzer: ">=4.0.0 <5.0.0"
+  analyzer: ">=5.1.0 <6.0.0"
   args: ">=2.1.0 <3.0.0"
-  dart_style: ^2.2.3
+  dart_style: ^2.2.4
   intl: ">=0.17.0 <0.20.0"
   path: ">=1.8.0 <2.0.0"
   petitparser: ">=5.0.0 <6.0.0"
 dev_dependencies:
   test: ^1.21.1
+dependency_overrides:
```

Comment:
that's should not be needed

Reply:
Unfortunately, I think this is kind of needed right now. If I remove this dependency override, we get this error:

```
[intl_generator] dart pub get
Resolving dependencies...
The current Dart SDK version is 2.17.7.

Because test >=1.21.5 requires SDK version >=2.18.0-146.0.dev <3.0.0 and test >=1.21.0 <1.21.5 depends on analyzer >=2.0.0 <5.0.0, test >=1.21.0 requires analyzer >=2.0.0 <5.0.0.
So, because intl_generator depends on both analyzer ^5.1.0 and test ^1.21.1, version solving failed.
exit code 1
```

But, the latest version of the `test` package requires Dart SDK 2.18.0+. So when I try to upgrade `test`, we get this error:

```
[intl_generator] dart pub get
Resolving dependencies...
The current Dart SDK version is 2.17.7.

Because intl_generator depends on test >=1.21.5 which requires SDK version >=2.18.0-146.0.dev <3.0.0, version solving failed.
exit code 1
```

This is because the latest stable version of Flutter uses Dart 2.17.7. So I think it would cause some trouble to try to upgrade `test` right now.

So, my recommendation is to keep this override for now since it is only overriding the transitive dependency on the `test` package, which is a dev dependency.